pkgs/by-name/om/omekasy/package.nix

```
@@ -0,0 +1,26 @@
+{
+  rustPlatform,
+  fetchFromGitHub,
+  lib,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "omekasy";
+  version = "1.2.3";
+
+  src = fetchFromGitHub {
+    owner = "ikanago";
+    repo = "omekasy";
+    rev = "v${version}";
+    hash = "sha256-oPhO+gRWrwgABc+gGXnIC519F5XVvewUHo2y54RoE4U=";
+  };
+
+  cargoSha256 = "sha256-6GjNn7FAcAihqNhPD18sUFe40ZQwXmFEQmoZNZL2trQ=";
```

```suggestion
  cargoHash = "sha256-6GjNn7FAcAihqNhPD18sUFe40ZQwXmFEQmoZNZL2trQ=";
```

I'm blocking it in staging https://github.com/NixOS/nixpkgs/pull/323983

---

I blame the build I stole from (lls) ^^.